apps/server/src/modules/files/entity/file-security-check.entity.ts

```diff
@@ -0,0 +1,42 @@
+import { Embeddable, Enum, Property } from '@mikro-orm/core';
+import { v4 as uuid } from 'uuid';
+
+import { FileSecurityCheckStatus } from '../domain';
+
+export interface FileSecurityCheckEntityProps {
+	status?: FileSecurityCheckStatus;
+	reason?: string;
+	requestToken?: string;
+}
+
+@Embeddable()
+export class FileSecurityCheckEntity {
```

Comment:
It's not just for the deletion as it was in the legacy code: we're also preparing a solution to retrieve the complete data because of the Initiative's business requirements. So to make it possible as well as having the old code regarding the data deletion migrated from the legacy part of the server, we need to keep the entities models complete. That's why we're also rewriting all the missing entity fields from the legacy code.